packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/brainTreeDropIn.js

```diff
@@ -40,40 +40,54 @@ const BraintreeDropin = props => {
     const [dropinInstance, setDropinInstance] = useState();
 
     const createDropinInstance = useCallback(async () => {
-        try {
-            const { default: dropIn } = await import('braintree-web-drop-in');
-            const dropinInstance = await dropIn.create({
-                authorization,
-                container: `#${containerId}`,
-                card: {
-                    cardholderName: {
-                        required: true
-                    },
-                    overrides: {
-                        fields: {
-                            number: {
-                                maskInput: {
-                                    // Only show last four digits on blur.
-                                    showLastFour: true
-                                }
+        const { default: dropIn } = await import('braintree-web-drop-in');
+        const dropinInstance = await dropIn.create({
+            authorization,
+            container: `#${containerId}`,
+            card: {
+                cardholderName: {
+                    required: true
+                },
+                overrides: {
+                    fields: {
+                        number: {
+                            maskInput: {
+                                // Only show last four digits on blur.
+                                showLastFour: true
                             }
                         }
                     }
                 }
-            });
-            setDropinInstance(dropinInstance);
+            }
+        });
+
+        return dropinInstance;
+    }, [containerId]);
+
+    useEffect(() => {
+        let unmounted = false;
+
+        const renderDropin = async () => {
+            const instance = await createDropinInstance();
+            setDropinInstance(instance);
```

Comment:
The gate is in the wrong place. You need to put the `if(!unmounted)` check here _within_ the async function. The bug is the same as your earlier race condition - attempting to set state on an unmounted component.